src/renderer/shared/ui/Alert/AlertItem.tsx

```diff
@@ -1,15 +1,17 @@
 import { PropsWithChildren } from 'react';
 
 import { FootnoteText } from '@renderer/shared/ui';
+import { cnTw } from '@renderer/shared/lib/utils';
 
 type Props = {
   withDot?: boolean;
+  className?: string;
 };
 
-const AlertItem = ({ withDot = true, children }: PropsWithChildren<Props>) => (
-  <li className="flex gap-x-1">
+const AlertItem = ({ withDot = true, children, className }: PropsWithChildren<Props>) => (
+  <li className={cnTw('flex gap-x-1', className)}>
     {withDot && <span className="shrink-0 w-1 h-1 rounded-full bg-text-secondary mt-[7px]" />}
-    <FootnoteText className="text-text-secondary tracking-tight">{children}</FootnoteText>
+    <FootnoteText className="text-text-secondary tracking-tight max-w-full">{children}</FootnoteText>
```

Comment:
Without this class something works wrong - `max-w-full`?
Same for `className`
I thought we got all required classes on `<li>` so nothing from outside world should alter it

Reply:
Yaeh, we need to set some width to properly handle overflow